routes.js

```diff
@@ -1,6 +1,11 @@
 module.exports = (app) => {
   const { register, login } = require("./controllers/user.controller.js");
-  const { all, add } = require("./controllers/doctor.controller.js");
+  const {
+    all: getAllDoctors,
```

Comment:
use the correct names of the controllers in files
https://medium.com/webbdev/7-%D1%80%D0%B5%D0%BA%D0%BE%D0%BC%D0%B5%D0%BD%D0%B4%D0%B0%D1%86%D0%B8%D0%B9-%D0%BF%D0%BE-%D0%BE%D1%84%D0%BE%D1%80%D0%BC%D0%BB%D0%B5%D0%BD%D0%B8%D1%8E-%D0%BA%D0%BE%D0%B4%D0%B0-%D0%BD%D0%B0-javascript-4d0d4c287281